lib/openstudio/lib/openstudio_aws_wrapper.rb

```diff
@@ -279,11 +322,13 @@ def load_private_key(filename)
     @private_key = File.read(filename)
   end
 
-  def save_private_key(filename)
+  def save_private_key(directory = '.', filename = 'ec2_server_key.pem')
     if @private_key
-      @private_key_file_name = File.expand_path filename
-      File.open(filename, 'w') { |f| f << @private_key }
-      File.chmod(0600, filename)
+      @private_key_file_name = File.expand_path "#{directory}/#{filename}"
+      logger.info "Saving server private key in #{@private_key_file_name}"
+      File.open(@private_key_file_name, 'w') { |f| f << @private_key }
+      logger.info 'Setting permissions of server private key to 0600'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.